### DISCUSSION TITLE
About OptiFine Format

### DISCUSSION THREAD

**Discussion by JLSkyzer** [MoreMcmeta/core#66] [Help Wanted]
Hey,
On emmissive page, you said `MoreMcmeta also supports the OptiFine format for emissive textures. Please see the [OptiFine documentation](https://github.com/sp614x/optifine/blob/master/OptiFineDoc/doc/emissive.properties) for details about this format`

but yes when we see doc we can see the file need to be on "assets/minecraft/optifine/emissive.properties" but you don't say if for your mod, the file need to be in the same path or no because i don't have optifine

**Comment by soir20** [comment]
The `emissive.properties` file does need to be on the `assets/minecraft/optifine/emissive.properties` path. Any emissive resource pack that works with OptiFine should also work with the MoreMcmeta emissive plugin, no changes needed. You also don't need OptiFine for the emissive plugin to work.

Please let me know if that doesn't fully answer your question.

**Reply by JLSkyzer** [reply]
ok but if i want to make all texture finish by _e are emissive is possible ? how ?

**Reply by soir20** [reply]
You would do exactly what you do with OptiFine, create an `emissive.properties` in the `assets/minecraft/optifine/` folder in a resource pack. The content of `emissive.properties` would be:

```properties
suffix.emissive=_e
```

It's perfectly valid to have a `assets/minecraft/optifine/` folder, even if OptiFine isn't installed.

You don't need to do anything differently to make it work with the MoreMcmeta emissive plugin. For example, you could take [this emissive pack](https://www.curseforge.com/minecraft/texture-packs/emissive), put it in your resource packs folder, and load the game with MoreMcmeta and the emissive plugin installed. No OptiFine necessary.
